service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseInitiationServiceTest.java

```diff
@@ -129,14 +130,21 @@ void shouldUpdateCaseDataWhenLocalAuthorityUserCreatesCase() {
                 .isEqualTo(LOCAL_AUTHORITY_NAME);
 
             assertThat(updatedCaseData.getLocalAuthorityPolicy())
-                .isEqualTo(organisationPolicy(LOCAL_AUTHORITY_ORG_ID, "[LASOLICITOR]"));
+                .isEqualTo(organisationPolicy(LOCAL_AUTHORITY_ORG_ID, null, "[LASOLICITOR]"));
 
             assertThat(updatedCaseData.getOutsourcingPolicy()).isNull();
         }
 
         @Test
         void shouldUpdateCaseDataWhenOutsourcedUserCreatesCase() {
-            givenUserFromOrganisation(EXTERNAL_ORG_ID);
+            Optional<Organisation> localOrganisation = Optional.ofNullable(LOCAL_AUTHORITY_ORG_ID)
```

Comment:
this is bad for multiple reason:
1. Making optional of non null static constant is not good, maybe:
            Organisation localOrganisation = Organisation.builder()
                .organisationIdentifier(LOCAL_AUTHORITY_ORG_ID)
                .name(LOCAL_AUTHORITY_NAME)
                .build();

            Organisation outsourcedOrganisation = Organisation.builder()
                .organisationIdentifier(EXTERNAL_ORG_ID)
                .name(EXTERNAL_ORG_NAME)
                .build();
            
            given(organisationService.findOrganisation()).willReturn(Optional.of(localOrganisation));
            given(organisationService.findOrganisation()).willReturn(Optional.of(outsourcedOrganisation));
            
2nd 
having 
given(organisationService.findOrganisation()).willReturn(org1);
given(organisationService.findOrganisation()).willReturn(org2);

is logically incorrect, organisationService.findOrganisation() give logged in user org, it cannot be that first call tells that user is in org1, and second in org2